packages/framework-core/src/decorators/command.ts

```diff
@@ -1,13 +1,13 @@
 import { Booster } from '../booster'
-import { Class, CommandInterface, RoleAccess } from '@boostercloud/framework-types'
+import { CommandClassInterface, RoleAccess } from '@boostercloud/framework-types'
 import { getPropertiesMetadata } from './metadata'
 
 /**
  * Annotation to tell Booster which classes are your entities
  * @param attributes
  * @constructor
  */
-export function Command(attributes: RoleAccess): (commandClass: Class<CommandInterface>) => void {
+export function Command(attributes: RoleAccess): <TCommand>(commandClass: CommandClassInterface<TCommand>) => void {
```

Comment:
Related to this comment LINK

If you change this to: 
```suggestion
export function Command(attributes: RoleAccess): (commandClass: CommandInterface) => void {
```
It will work as expected and there is no need for the `CommandClassInterface`  😃

Reply:
I don't think so, because we are expecting that the class itself (not the instances) to have that interface, so the passed object must be of that type. Check out https://stackoverflow.com/questions/13955157/how-to-define-static-property-in-typescript-interface

Reply:
Have you tried it? Check this [playground](https://www.typescriptlang.org/play?#code/FASwdgLgpgTgZgQwMZQAQGEA2CDOOA8AKgEpRyZRLQAmAfKgN7CqphQDuqAFAHR8IwA5jgBcqBGACeAbQC6ASjEkyFKlGrAAvsGAB6XagC0xk6bPmLJnUmx5UAdRAQAFgGUICCCCQBZKC4B7akZmFlQABwBXACNMb1QcDy8kVGcJagouRVQANwCQYKYw4tQkALAcAIoeTADBLgByRM9vBvlQlm1tYBtcHAcnZwBJCo8wFD9Awo6ImLiUtLAMqCyxPIKQkrCyiqqoGrrG8GbxqDaZrp1waHhkNAAJdIoYTbDF5dXc-I1u-SNLAGA0w6OCRcZecqoSI4KA4dwtJCPJbPLjvZ5iJHLGDyV4sNGwHj4rJaEFgqggSHQ2FYPr2ZwBGEjE4oIY4TEo-EwMQ0vD4dmwWg4ophNicTlZQlPFbtbo6KlwpLefkwLiOFzw5KTenUdp-eUapVSlVq4ajCQTfzanEAEQCsLADQgpQCAFtwiAKHKYTgeTg6QyoEyxiy2UbVYMg+aoFqgrqDPLff7GWbTqzleH1YrfJbY6hbfbHc63R6oEA)

Reply:
Doh! I didn't reallize that I duplicated the responsibility of `CommandInterface` and renamed it. Now I understood, thanks for pointing it out. Fixed in https://github.com/boostercloud/booster/pull/333/commits/47bb8bfdefe0210d9e7efe72065084d714e5ab58